University-advisor-web/Controllers/RegistrationController.cs

```diff
@@ -12,6 +12,11 @@ public IActionResult Index()
             return View(model);
         }
 
+        //public IActionResult Display()
+        //{
+        //    return View("../Registration/Index");
+        //}
+
```

Comment:
I guess if that is not needed it can be deleted.

Reply:
That is yours code, but I will delete it.

Reply:
Deleted.